lua/nvim-treesitter/parsers.lua

```diff
@@ -2101,6 +2102,48 @@ list.supercollider = {
   maintainers = { "@madskjeldgaard" },
 }
 
+list.superhtml = {
+  install_info = {
+    url = "https://github.com/kristoff-it/superhtml",
+    includes = { "tree-sitter-superhtml/src" },
+    files = {
+      "tree-sitter-superhtml/src/parser.c",
+      "tree-sitter-superhtml/src/scanner.c",
+    },
+    branch = "main",
+  },
+  filetype = "superhtml",
+  maintainers = { "@rockorager" },
+}
+
+list.supermd = {
```

Comment:
The metadata of this repo's package.json is still that of `tree-sitter-markdown`. You should change it.

Addtionall question: The grammar code seems to have a lot of duplicates with Markdown parser's one. You can probably extend from it, some examples can be:
- https://github.com/tree-sitter-grammars/tree-sitter-vue/blob/main/grammar.js
- https://github.com/tree-sitter/tree-sitter-cpp/blob/master/grammar.js
- https://github.com/tree-sitter/tree-sitter-typescript/blob/master/common/define-grammar.js

Reply:
I can also recommend https://github.com/tree-sitter-grammars/template for a repo layout and workflows. It's very important to have grammars be as consistent as possible for editors to be able to include them.

Reply:
I'll see if upstream will consider extending the existing markdown grammar. Is this a dealbreaker for inclusion?

Reply:
Not really; we care more about the queries (and that `tree-sitter generate` and `tree-sitter build` work out of the box). But it's a good suggestion, and _if_ it is followed may lead to possibly breaking changes in the queries -- and it would be good to avoid that (i.e., wait until the "final" queries can be added here).

Reply:
(But another indication why separate PRs for separate languages is a good idea ;))

Reply:
> The metadata of this repo's package.json is still that of `tree-sitter-markdown`. You should change it.

https://github.com/kristoff-it/supermd/pull/1